### PR TITLE
Login Page

### PR DESCRIPTION
**Pull request by ydidwania**
Added a button to skip login. 
Created HomeAcrivity, which is only a Welcome page as of now and displays user_id if  logged in  otherwise "Guest"
Addresses #2

### CODE REVIEW

File: app/src/main/java/com/iitb/ferozepurwale/lendit_app/HomeActivity.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+public class HomeActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home);
+        TextView nameView = (TextView)findViewById(R.id.name);
+        Bundle from_main = getIntent().getExtras();
+        boolean logged_in = from_main.getBoolean("logged_in");;
```
Comment (TheFlash98) [inline app/src/main/java/com/iitb/ferozepurwale/lendit_app/HomeActivity.java]: Why are there double`;;`. Is it supposed to be this way?

Comment (ydidwania) [reply app/src/main/java/com/iitb/ferozepurwale/lendit_app/HomeActivity.java]: Forgive me lord for i have sinned

Comment (TheFlash98) [reply app/src/main/java/com/iitb/ferozepurwale/lendit_app/HomeActivity.java]: Uhm? Did I say anything wrong? 😅

Comment (CodeMaxx) [reply app/src/main/java/com/iitb/ferozepurwale/lendit_app/HomeActivity.java]: @TheFlash98 Explain me what this statement > `;` does

Comment (TheFlash98) [reply app/src/main/java/com/iitb/ferozepurwale/lendit_app/HomeActivity.java]: Marks the end of the expression?

Comment (CodeMaxx) [reply app/src/main/java/com/iitb/ferozepurwale/lendit_app/HomeActivity.java]: Ignore maar

Comment (ydidwania) [reply app/src/main/java/com/iitb/ferozepurwale/lendit_app/HomeActivity.java]: sorted now :)

**Comment by nihal111** [comment]
Gave it a quick test. Found the following problems-
1. Logging in and logging out worked well the first time. Subsequent logins gave this-
screenshot_2017-08-27-22-15-26-359_com facebook katana

2. `FacebookSdk.sdkInitialize(getApplicationContext());` is deprecated. Reevaluate if this is necessary.
3. I would prefer if all variables of the form `new_variable` are renamed to `newVariable` following the Java capitalisation standard elsewhere.

**Comment by TheFlash98** [comment]
This shouldn't come unless @ydidwania changed something in my code.

File: app/build.gradle [hunk 2]
```diff
@@ -19,7 +19,9 @@ android {
     }
 }
 repositories {
-    mavenCentral()
+    jcenter {
```
Comment (ydidwania) [inline app/build.gradle]: I was getting an error `Failed to resolve: com.facebook.android:facebook-android`. 
Made the neccessary [change](https://stackoverflow.com/questions/31798984/failed-to-resolve-com-facebook-androidfacebook-android-sdk4-4-0)

**Comment by ydidwania** [comment]
I added mavenCentral() to undo the change. 
@nihal111 could you test again and make sure everything is fine?

**Comment by TheFlash98** [comment]
@ydidwania did you try what @nihal111 did? Multiple login and logouts?

**Comment by nihal111** [comment]
Tested out again. Works. Merging.
I'm not sure what the TextView in the MainActivity is meant for. Most of the usages are commented out. Make sure to remove that later.

**Pull request merged**